plugins/src/main/java/com/google/fhir/gateway/plugin/OpenSRPHelper.java

```diff
@@ -0,0 +1,532 @@
+package com.google.fhir.gateway.plugin;
+
+import static org.smartregister.utils.Constants.EMPTY_STRING;
+
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.gclient.ReferenceClientParam;
+import com.google.fhir.gateway.ProxyConstants;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.r4.model.BaseResource;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.CareTeam;
+import org.hl7.fhir.r4.model.Enumerations;
+import org.hl7.fhir.r4.model.Group;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Location;
+import org.hl7.fhir.r4.model.Organization;
+import org.hl7.fhir.r4.model.OrganizationAffiliation;
+import org.hl7.fhir.r4.model.Practitioner;
+import org.hl7.fhir.r4.model.PractitionerRole;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.smartregister.model.location.LocationHierarchy;
+import org.smartregister.model.location.ParentChildrenMap;
+import org.smartregister.model.practitioner.FhirPractitionerDetails;
+import org.smartregister.model.practitioner.PractitionerDetails;
+import org.smartregister.utils.Constants;
+import org.springframework.lang.Nullable;
+
+public class OpenSRPHelper {
+  private static final Logger logger = LoggerFactory.getLogger(OpenSRPHelper.class);
+  public static final String PRACTITIONER_GROUP_CODE = "405623001";
+  public static final String HTTP_SNOMED_INFO_SCT = "http://snomed.info/sct";
+  public static final Bundle EMPTY_BUNDLE = new Bundle();
+  private IGenericClient r4FHIRClient;
```

Comment:
For acronyms we don't capitalize all letters, only the first
```suggestion
  private IGenericClient r4FhirClient;
```

Reply:
Noted